app/views/event_mailer/new_event.html.erb

```diff
@@ -7,4 +13,11 @@
   When: <%= @event.start_time %>
 </blockquote>
 
-<p>To see more details about the event, please visit the event page.</p>
+<p>To see more details about the event, please visit the <%= link_to "event page",
+        :controller => "events",
+        :action => "show",
+        :only_path => false,
+        :campus_id => @room.campus.id,
+        :room_id => @event.room.id,
+        :id => @event.id,
+        :host => "example.com" %>.</p>
```

Comment:
Why is this example.com? Shouldn't it be pulling from the `development.rb` and `production.rb` configuration that we set?